examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowClient.java

```diff
@@ -65,6 +65,26 @@ public static void main(String[] args) throws InterruptedException {
         System.out.printf("waitForInstanceCompletion has an exception:%s%n", ex);
       }
 
+      System.out.println(separatorStr);
+      System.out.println("**purgeInstance**");
+      try {
+        boolean purgeResult = client.purgeInstance(instanceId);
+        System.out.printf("purgeResult: %s%n",purgeResult);
+      } catch (Exception ex) {
+        System.out.printf("purgeInstance has an exception:%s%n", ex);

```

Comment:
I think you don't need try/catch here, because the method not expecting to raise an exception and it is not defined in its signature